app/src/main/res/layout/row_issue_create_label.xml

```diff
@@ -21,11 +22,10 @@
         android:layout_height="wrap_content"
         android:foreground="?attr/selectableItemBackground"
         android:paddingBottom="5dp"
-        android:paddingLeft="8dp"
+        android:paddingLeft="16dp"
         android:paddingRight="8dp"
         android:paddingTop="5dp"
         android:textAppearance="?android:attr/textAppearanceMedium"
-        app:ghFont="condensed"
```

Comment:
With that change the label selection UI is now inconsistent to the label management UI ... I'd prefer to keep this.

Reply:
I don't really agree on this, but I don't mind reverting this change if you have a strong opinion.
To keep consistency with the other screen, I'd also consider changing the label management UI not to use condensed font, which together with list separators gives it a quite dated look IMO.

Reply:
As long as both are consistent, I don't have an overly strong opinion either way.

Reply:
Reverted this change for now to keep UIs consistent and unblock the other changes in this PR